scripts/globals/items/qiqirn_mine.lua

```diff
@@ -0,0 +1,27 @@
+-----------------------------------------
+-- Qiqirn Mine
+-- ID 5331
+-- Summons a mine to blowup
+-----------------------------------------
+local ID = require("scripts/zones/Lebros_Cavern/IDs")
+require("scripts/globals/status")
+require("scripts/globals/zone")
+-----------------------------------------
+
+function onItemCheck(target)
+    local result = 0
+    if target:getZoneID() == LEBROS_CAVERN then
+        result = 308
+    end
+    return result
+end
+
+function onItemUse(target,npc,mob)
```

Comment:
you can't just make up parameters. look at the definition of onItemUse in the core.